src/utils/response.ts

```
@@ -3,13 +3,86 @@ export const mergeResponseInit = (
   responseInit?: ResponseInit,
 ): ResponseInit => {
   const headers = new Headers(defaultHeaders);
-
   const override = responseInit?.headers;
   if (override) {
-    new Headers(override).forEach((value, key) => {
-      headers.set(key, value);
-    });
+    new Headers(override).forEach((value, key) => headers.set(key, value));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)